### ISSUE TITLE
Django 1.10.1 + MULTI-LINGUAL SITES "manage.py createdb" => error

### ISSUE THREAD

**Issue by webzone-vn** [stephenmcd/mezzanine#1679]
Base on doc: http://mezzanine.jupo.org/docs/multi-lingual-sites.html
With django 1.8.15 everything is OK.
With Django 1.10.1 when I run manage.py createdb it show errors:
![manage.py createdb](http://i.imgur.com/qmZYIl8.png)

**Comment by stephenmcd** [comment]
I just pushed c244b603a6efab5062dcf97b1e12227e61ba4fb8 which fixes the error, but now a new error is raised that looks to be within django-modeltranslation. I don't have any understanding of it myself.

**Comment by cnbillow** [comment]
django 1.10.2
Mezzanine 4.2.2

`Traceback (most recent call last):
  File "..\homepage\manage.py", line 14, in <module>
    execute_from_command_line(sys.argv)
  File "..\lib\site-packages\django\core\management\__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "..\lib\site-packages\django\core\management\__init__.py", line 359, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "..\lib\site-packages\django\core\management\base.py", line 294, in run_from_argv
    self.execute(*args, **cmd_options)
  File "..\lib\site-packages\django\core\management\base.py", line 345, in execute
    output = self.handle(*args, **options)
  File "..\lib\site-packages\modeltranslation\management\commands\update_translation_fields.py", line 32, in handle
    model._default_manager.filter(q).rewrite(False).update(
AttributeError: 'SearchableQuerySet' object has no attribute 'rewrite'`

**Comment by logileifs** [comment]
same thing happening to me, I'm unable to create a new multi lingual project and unable to make an existing project multi lingual.

Probably related to this one: https://github.com/deschler/django-modeltranslation/issues/389

It would be very nice to get a fix as soon as possible

**Comment by mwort** [comment]
Same here:
Mezzanine==4.2.2
django-modeltranslation==0.12

$ python manage.py update_translation_fields
```
Using default language: de
Updating data of model '<class 'cartridge.shop.models.ProductVariation'>'
Updating data of model '<class 'cartridge.shop.models.Category'>'
Traceback (most recent call last):
  File "manage.py", line 14, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/michel/Desktop/klipklap/pythonenv/lib/python2.7/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/Users/michel/Desktop/klipklap/pythonenv/lib/python2.7/site-packages/django/core/management/__init__.py", line 359, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/michel/Desktop/klipklap/pythonenv/lib/python2.7/site-packages/django/core/management/base.py", line 294, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/michel/Desktop/klipklap/pythonenv/lib/python2.7/site-packages/django/core/management/base.py", line 345, in execute
    output = self.handle(*args, **options)
  File "/Users/michel/Desktop/klipklap/pythonenv/lib/python2.7/site-packages/modeltranslation/management/commands/update_translation_fields.py", line 32, in handle
    model._default_manager.filter(q).rewrite(False).update(
AttributeError: 'SearchableQuerySet' object has no attribute 'rewrite'
```
Would be great if a solution could be found soon.

**Comment by PapugazLasu** [comment]
Oh, I've got the same AttributeError. Any help?

**Comment by rubencabrera** [comment]
Another one bites the dust here.

**Comment by rubencabrera** [comment]
In case it helps anyone else, I just tried the fix by @melvyn-sopacua linked from the issue linked by @logileifs and it let me go on with migrations, so seems to work quite fine (as long as I have tested).

**Comment by bigoss** [comment]
I have applied above patch.
However while adding form page in admin i get:

__
    self.fields["label"].widget = admin.widgets.AdminTextInputWidget()
KeyError: u'label'
[12/Jun/2017 10:45:05] "GET /en/admin/forms/form/add/ HTTP/1.1" 500 149820

Can anyone help please?

**Comment by stephenmcd** [comment]
> self.fields["label"].widget = admin.widgets.AdminTextInputWidget()
> KeyError: u'label'

Should be fixed in a0283578be093840558b09dee97bafbb5c925841
